features/rerun_profile.feature

```diff
@@ -26,15 +26,15 @@ Feature: Rerun profile
     And a file named "features/step_definitions/rerun_steps.rb" with:
       """
       Given('fixed now') do
-        puts 'All fixed now'
+        log 'All fixed now'
```

Comment:
Resolve the Cucumber deprecation warnings:


> WARNING: #puts(message) is deprecated and will be removed after version 6.0.0. Messages emitted with "puts" will no longer be caught by Cucumber and sent to the formatter. If you want message to be in the formatted output, please use log(message) instead.

Reply:
Oooh, this is awesome, this will fix the CI errors in #496, no? 😄 
Huge +1 for this and thanks for taking care as well! 🎉